app/controllers/health_checks_controller.rb

```diff
@@ -0,0 +1,12 @@
+class HealthChecksController < ApplicationController
+  def index
+    london_ips = ENV.fetch('LONDON_RADIUS_IPS').split(',')
+    dublin_ips = ENV.fetch('DUBLIN_RADIUS_IPS').split(',')
+
+    health_checks_fetcher_use_case = UseCases::Administrator::HealthChecks::Fetcher.new(ips: london_ips + dublin_ips)
+
+    @health_checks = UseCases::Administrator::HealthChecks::CalculateHealth.new(
+      health_checks_fetcher: health_checks_fetcher_use_case
+    ).execute
```

Comment:
This second Use Case seems like a presenter to me i.e its only job is to prepare previously fetched data for display on the page. Could we pass it in as a dependency to the first use case `.execute` method? On the presenter itself you can just have a method like `.present` to get the data in a format you want.

Example: https://github.com/alphagov/govwifi-user-signup-api/blob/master/tasks/publish_statistics.rb#L13

Reply:
In general, second Use Case does not look like a specific business need for me.